jerry-main/main-unix.c

```diff
@@ -193,7 +193,7 @@ main (int argc,
     }
     else if (!strcmp ("-v", argv[i]) || !strcmp ("--version", argv[i]))
     {
-      jerry_port_console ("Version: \t%d.%d\n\n", JERRY_API_MAJOR_VERSION, JERRY_API_MINOR_VERSION);
+      jerry_port_console ("Version: %d.%d%s\n", JERRY_API_MAJOR_VERSION, JERRY_API_MINOR_VERSION, JERRY_COMMIT_HASH);
```

Comment:
The space is added by CMakeLists, if needed. If the build is performed in a git repo, `JERRY_COMMIT_HASH` becomes " (xxxxxxx)", but if .git dir does not exist, then it's an empty string.